src/commands/ExecCommand.js

```diff
@@ -18,17 +18,19 @@ export const builder = {
     group: "Command Options:",
     describe: "Bail on exec execution when the command fails within a package",
     type: "boolean",
-    default: true,
+    default: undefined,
```

Comment:
Yeah, I didn't like having to do it that way. Maybe something Yargs could do here (as a breaking change), would be to default to `undefined` instead of `false`. Other than that, this doesn't feel like an issue with Yargs, but more of how Lerna is using their defaults in conjunction with other forms of obtaining options that are merged in with them. The only alternative would be for Lerna to insert its durable options in between user options that yargs parses and their defaults they automatically merge in.

Do you think it's worth raising an issue for one of these for Yargs and referencing this PR?